Add support for (non-)legit workday of China

EraserKing opened the pull request:
This pull request aims to support repeat option of (non-)legit workdays in Clock app, by bypassing international build detect.
Verified on MIUI EU 8.9.20 / MIX 2S.

I don't know much about bash shell so it may look ugly and need some polish. Thanks.

EraserKing commented on cleaner-fix.sh:
```diff
@@ -231,8 +263,14 @@ extract() {
+        fi
     done
+    mv deodex/system/app/DeskClock deodex/system/priv-app/DeskClock
+    rm -r deodex/system/app
```
Because the other places are always dealing with apps in priv-app folder, so I also move DeskClock.app from app folder into priv-app folder. This would eventually leads to a side effect: duplicated apps in both app and priv-app folders after flashing the zip package, but it seems has no harm.

linusyang92 commented:
@EraserKing Hi, thanks for the patch. But currently I have no plan to modify the clock app. I am not sure if this patch will bring any side effects. I'll leave this PR open.

EraserKing commented:
Hi, I have rewritten my commit, and now it allows to add some regular apps (/data/app) to be modified, plus the original way to update the priv-apps (/data/priv-app).
As an example, I added a command line switch to edit the clock app, and by default it's not turned on.

Additionally, to reduce file size, I created a new method to edit the international flag and modified the related methods to call this new method instead, to avoid duplicated modifications.
